addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -241,12 +241,12 @@ $input-border-radius: o-website-value('input-border-radius') !default;
 $input-border-radius-lg: o-website-value('input-border-radius-lg') !default;
 $input-border-radius-sm: o-website-value('input-border-radius-sm') !default;
 
-$input-bg: var(--o-input-bg, #{$light}) !default;
-$input-border-color: var(--o-input-border-color, #{$light}) !default;
-$input-color: color-contrast($light) !default;
-$input-focus-bg: $body-bg !default;
-$input-focus-color: $body-color !default;
-$input-placeholder-color: var(--o-input-placeholder-color, #{rgba($input-color, .5)}) !default;
+$input-bg: o-website-value('input-background-color') or o-color('o-cc1-bg') !default;
```

Comment:
Nitpicking: use `$body-bg` instead of `o-color('o-cc1-bg')`, more explicit once `$body-bg` has already been set`.